cmd/marker/main.go

```diff
@@ -0,0 +1,24 @@
+package main
```

Comment:
Can't leave comment for a binary file, but look below, you included cmd/marker/marker binary into the patch. Please remove.